projects/packages/my-jetpack/_inc/components/icons/stories/index.jsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import * as allIcons from '../index.jsx';
+import styles from './style.module.scss';
+
+export default {
+	title: 'Packages/My Jetpack/Icons',
+	component: allIcons,
+	parameters: {},
+};
+
+const sizes = [
+	{
+		label: 'small',
+		value: '24',
+	},
+	{
+		label: 'medium',
+		value: '48',
+	},
+	{
+		label: 'large',
+		value: '72',
+	},
+];
+
+/**
+ * Icons story components.
+ *
+ * @returns {object} - story component
+ */
+function IconsStory() {
+	return (
+		<div>
+			{ sizes.map( size => (
+				<div key={ size.label }>
+					<h3>{ size.label }</h3>
+					<div className={ styles[ 'icons-container' ] }>
+						{ Object.keys( allIcons ).map( key => {
+							const Icon = allIcons[ key ];
+							return (
+								<div
+									className={ `${ styles[ 'icon-wrapper' ] } ${ styles[ size.label ] }` }
+									key={ key }
+								>
+									<Icon size={ size.value } />
+									<span>{ Icon.displayName.replace( /icon/gi, '' ) }</span>
```

Comment:
I'm not sure I understand why are we doing this. Looks super hacky

Reply:
it's the stories. It just iterates along all icons to render them. I don't think we need to concern care too much about it.

Reply:
👍 I meant the replace thing...

Reply:
ohhh, yeah. Well, I did it to get the Icon name from the product class. We could add a `titleComponentMap`, or change the displayName ... but I don’t know 🤷‍♂️